### ISSUE TITLE
Feature Request: Add to bpkg for easy install.

### ISSUE THREAD

**Issue by jglenn9k** [bats-core/bats-core#234]
http://www.bpkg.sh/submit/

**Comment by rodrigobdz** [comment]
This feature seems to be already implemented: https://github.com/bats-core/bats-core/wiki/Install-Bats-Using-a-Package#installation-with-bpkg

**Comment by sublimino** [comment]
True, but it's not currently in http://www.bpkg.sh/packages/name/ -- some investigation required D:
